File: src/main/java/personal/alexgarland/helloworldmvc/service/IEmployeeRepository.java
package personal.alexgarland.helloworldmvc.service;

import java.util.List;
import personal.alexgarland.helloworldmvc.model.Employee;

public interface IEmployeeRepository {

  public List<Employee> getEmployeeList();

  public Employee addEmployee(Employee e);

  public void deleteEmployeeById(int employeeId);

  public void updateEmployee(Employee e);

  public Employee getEmployeeById(int employeeId);

}

File: src/main/kotlin/personal/alexgarland/helloworldmvc/service/repository/IEmployeeRepository.kt
package personal.alexgarland.helloworldmvc.service.repository

import personal.alexgarland.helloworldmvc.model.Employee

interface IEmployeeRepository {

    val employeeList: List<Employee>

    fun addEmployee(e: Employee): Employee

    fun deleteEmployeeById(employeeId: Int)

    fun updateEmployee(e: Employee)

    fun getEmployeeById(employeeId: Int): Employee

}
